release-notes.md

```diff
@@ -0,0 +1,299 @@
+# Release Notes
+
+These release notes are manually compiled from pull requests and Jira issues
+starting with CiviCRM 4.7.14.
+
+## CiviCRM 4.7.14
+
+Released December 2016
+
+### Credits
+
+This release was developed by the following code authors:
+
+AGH Strategies - Andrew Hunt; Australian Greens - Seamus Lee; CiviCRM - Coleman Watts, Jitendra Purohit, Monish Deb, Tim Otten, and Yashodha Chaku; CiviDesk - Nicolas Ganivet; CiviFirst - John Kirk; Coop SymbioTIC - Samuel Vanhove; Francesc Bassas i Bullich; Fuzion - Chris Burgess and Eileen McNaughton; Ginkgo Street Labs - Tobias Lounsbury; JMA Consulting - Edzel Lopez and Pradeep Nayak; Lighthouse Consulting and Design - Brian Shaughnessy; Progressive Technology Project - Jamie McClelland; Spry Digital - Ellen Hendricks; Systopia - Björn Endres; Third Sector Design - Michael McAndrew; Thomas Schüttler
```

Comment:
It's missing because I've only gotten through half of the PRs for this release.  I'll format it as such when I add you all along with the other contributors.